test/runtests.jl

```diff
@@ -1,7 +1,10 @@
+using Documenter
 using IndexedDims
 using Test
 
 @testset "IndexedDims.jl" begin
+    doctest(IndexedDims)
```

Comment:
Do these run in their own testset? They should.

Reply:
yes, with default name "Doctests", but we can set that with a keyword like `testset="Docstring examples"`
https://juliadocs.github.io/Documenter.jl/dev/lib/public/#Documenter.doctest